text/trace/0220-messaging-semantic-conventions-span-structure.md

```diff
@@ -126,19 +126,22 @@ sending or publishing to an intermediary. This call or operation (and the
 related "Publish" span) can either refer to a single message or to a batch of
 multiple messages.
 
-It is recommended to create a "Create" span for every single message. "Create"
-spans can be created during the "Publish" operation as children of the
-"Publish" span. Alternatively, "Create" spans can be created independently of
-the "Publish" operation, e. g. in cases where messages are created before they
-are passed to a "Publish" operation. In this case, the "Publish" span should
-link to the "Create" spans.
-
-If a "Create" span exists for a message, its context must be injected into the
-message. If no "Create" span exists for a message, the context of the related
-"Publish" span must be injected into the message.
-
-"Create" spans must not be created for messages into which a context has
-already been injected.
+There are three different scenarios for injecting a creation context into a message:
+
+1. A user provides a custom creation context for a message, which doesn't refer
+   to any span described in this document. This provides flexibility to users
+   to model custom scenarios. In this case, no other spans besides the "Publish"
+   span should be created.
+2. If no custom creation context is provided for a message, it is recommended
+   to create a "Create" span for every single message and inject its context.
```

Comment:
```suggestion
   to create a "Create" span for every single message and inject its context in the message.
```

Not sure if it's just me, but I had to read it 3x until I got "its context" was the span context in the message.

Reply:
Applied, just using "into" instead of "to".